setup.cfg

```
@@ -0,0 +1,54 @@
+# Minimum target setuptools 39.2.0
+
+[metadata]
+name = ansible-core
+version = attr: lib.ansible.release.__version__
+description = Radically simple IT automation
+long_description = file: README.rst
```

Wild idea: why not add some changelog pointers too?
```suggestion
long_description = file: README.rst, docs/docsite/rst/porting_guides/porting_guide_core_2.13.rst
```
Here's an example project doing this: https://pypi.org/project/attrs/#release-information.